src/vs/workbench/common/theme.ts

```diff
@@ -449,6 +449,12 @@ export const MENU_SELECTION_BORDER = registerColor('menu.selectionBorder', {
 	hc: null
 }, nls.localize('menuSelectionBorder', "Border color of the selected menu item in menus."));
 
+export const MENUBAR_SEPARATOR_BACKGROUND = registerColor('menu.separatorBackground', {
+	dark: '#BBBBBB',
+	light: '#888888',
+	hc: contrastBorder
+}, nls.localize('menuSeparatorBackground', "Color of a separator menu item in the menubar."));
```

Comment:
This should read `"Color of a separator menu item in menus."` as it applies to all menus.